src/content/docs/pl/getting-started.mdx

```diff
@@ -2,108 +2,63 @@
 title: Wprowadzenie
 description: Podstawowy wstęp do Astro.
 i18nReady: true
+tableOfContents: false
+editUrl: false
```

Comment:
There is this banner present on main:
```
banner:
  content: |
    Join us and 100+ Astro developers at <a href="https://lu.ma/astro-together-meetup-2024/">Astro Together</a> in Montreal on May 28th!
```

Should we include that? 🤔

Reply:
I didn't want to include that since it'll no longer be valid in about two weeks and I've seen other translations with even older ones still present. It could become misleading